src/urdf_renderer.cpp

```diff
@@ -85,16 +90,18 @@ namespace realtime_urdf_filter
     V_Link::iterator it = links.begin();
     V_Link::iterator end = links.end();
 
+    bool no_error = true;
     for (; it != end; ++it)
-      process_link (*it);
+      no_error = process_link(*it) && no_error;
```

Comment:
do we need to process all links? otherwise we could return false much earlier

Reply:
i thought it might be useful to print all the missing links

Reply:
true, that seems helpful!